src/sensors/bno080sensor.cpp

```diff
@@ -82,6 +82,20 @@ void BNO080Sensor::motionSetup()
 
 void BNO080Sensor::motionLoop()
 {
+#if ENABLE_INSPECTION
+    {
+        (void)imu.getReadings();
+
+        float rX, rY, rZ, aX, aY, aZ, mX, mY, mZ;
+        uint8_t rA, aA, mA;
+        imu.getGyro(rX, rY, rZ, rA);
```

Comment:
This will not work, you need to ask sensor to send this data during initialization in motionSetup()